src/yosys_mau/task_loop/logging.py

```diff
@@ -117,16 +144,28 @@ class LogContext:
     When set, the downgrading happens before the `LogEvent` is emitted.
     """
 
-    level: Level = "info"
-    """The minimum log level to display/log.
+    # context manager doesn't like properties that get passed by reference?
+    __levels: LevelDict | None = None
 
-    This does not stop `LogEvent` of smaller levels to be emitted. It is only used to filter which
-    messages to actually print/log. Hence, it does not affect any user installed `LogEvent`
-    handlers.
+    @property
+    def levels(self) -> LevelDict:
+        if not self.__levels:
+            self.__levels = LevelDict()
```

Comment:
I tried to have `levels = LevelDict()` but then the value was persisting between tests.  This is a work around to avoid that, but there's probably something in the context manager that should be doing deep copies instead?